tests/test_pipeline_images.py

```diff
@@ -169,7 +169,7 @@ def test_get_images(self):
 
         path, new_im, new_buf = next(get_images_gen)
         assert path == "full/3fd165099d8e71b8a48b2683946e64dbfad8b52d.jpg"
-        assert orig_im == new_im
+        assert orig_im.copy() == new_im
```

Comment:
`orig_im` would be the opened image in the specific class of `JpegImagePlugin` - https://github.com/scrapy/scrapy/actions/runs/16583228546/job/46903534162#step:5:262

Calling [`copy()`](https://pillow.readthedocs.io/en/stable/reference/Image.html#PIL.Image.Image.copy) turns it into a more general `Image` instance, to match what is returned by [`ImageOps.exif_tranpose()`](https://pillow.readthedocs.io/en/stable/reference/ImageOps.html#PIL.ImageOps.exif_transpose)